frontend/src/component/insights/components/LineChart/LineChartComponent.tsx

```diff
@@ -100,16 +105,18 @@ const LineChartComponent: FC<{
     const { locationSettings } = useLocationSettings();
 
     const [tooltip, setTooltip] = useState<null | TooltipState>(null);
+
     const options = useMemo(
-        () => ({
-            ...createOptions(
-                theme,
-                locationSettings,
-                setTooltip,
-                Boolean(cover),
-            ),
-            ...overrideOptions,
-        }),
+        () =>
```

Comment:
Replaced spreading with deepmerge. That allows me to pinpoint override properties deep in object tree .